future_challenges_for_RSEng.tex

```diff
@@ -310,22 +310,20 @@ \subsection{Impact of AI and automation}
 RSEs understand how AI models work, and can help researchers reap their benefits, while
 clearly knowing their limits.
 \end{whatis}
-AI is already transforming research processes by enabling rapid analysis of massive datasets, uncovering patterns and insights that would be impossible or prohibitively time-consuming for humans alone to find.
-In research and development, AI accelerates scientific breakthroughs in fields like healthcare, climate science, and materials engineering by automating complex modelling and predictive tasks.
-For software development, AI-powered tools automate code generation, bug detection, and testing, significantly increasing productivity and reducing human error.
-Developers are shifting from manual coding to orchestrating AI-driven workflows, focusing more on high-level design, integration, and creative problem-solving while AI handles repetitive or routine tasks.
-The integration of AI is democratizing access to advanced technologies and analytics, enabling non-experts to participate in research and software creation, and fostering greater collaboration across disciplines.
-As AI continues to evolve, it will further blur the lines between human and machine roles, making research and software development faster, more efficient, and more innovative than ever before.
-With the general availability of AI the interaction with researchers will change at a fundamental level.
+Generative AI tools, such as GitHub Copilot \cite{Friedmann2021}, are being added to software development tools.
+These AI powered code assistants promise to increase productivity by automatically generating code, and writing test cases and documentation.
+Software developers use them to reduce the number of keystrokes, complete tasks and to recall syntax without having to consult the documentation \cite{Liang2024}.
+Code assistants have the potential to help novice or even non-expert programmers write code \cite{Feldman2024} thus enabling non-experts to participate in research and software creation, and fostering greater collaboration across disciplines.
+However, when these tools are used in practice many users report increased cognitive load and frustration stemming from the difficulty to prompt the system effectively and debugging the generated code \cite{Simkute2025}.
+
+The potential of AI to be a democratising technology can only be realised when the algorithmic divide along with the digital divide is addressed \cite{Yu2020}.
```

Comment:
Is "democratising" the correct term and form here?

Is a separate paragaph needed?

Reply:
I think the term is ok. It was used before as well. I was wondering about the paragraph. Happy to remove that